.python-version

```diff
@@ -1 +1 @@
-3.5.6
+3.6.5
```

Comment:
This is only used by PyEnv, which can install the more modern `3.6.11` just as well.

Reply:
updated to 3.6.12 released in August

Reply:
Python 3.6.12 is not yet availabe via Homebrew's PyEnv, but that should be fixed by the time this lands. I just manually switched to 3.6.11 for testing. https://github.com/pyenv/pyenv/pull/1680#issuecomment-701478171